spec/std/socket/tcp_server_spec.cr

```diff
@@ -136,4 +136,18 @@ describe TCPServer, tags: "network" do
       end
     end
   {% end %}
+
+  describe "accept" do
+    {% unless flag?(:win32) %}
+      it "sets close on exec flag" do
+        TCPServer.open("localhost", 0) do |server|
+          TCPSocket.open("localhost", server.local_address.port) do |client|
+            server.accept? do |sock|
+              sock.close_on_exec?.should be_true
+            end
+          end
+        end
+      end
+    {% end %}
```

Comment:
**suggestion:** Maybe we could still run this test on Windows, but make the expectation `should eq {{ !flag?(:win32) }}`?

Reply:
I wondered about that, but the flag has no meaning on Windows, so the test would have no purpose, and the assertion would become weird, so I didn't.